launchpad/reward_calculation.gno

```diff
@@ -127,15 +146,21 @@ func (self *RewardState) finalize(currentHeight uint64) {
 		// Not started yet
 		return
 	}
+	if self.LastHeight >= self.EndHeight {
+		// already ended
+		return
+	}
 	if currentHeight > self.EndHeight {
 		currentHeight = self.EndHeight
 	}
 
+	// ????? height 차이에 의한 오차 고
```

Comment:
handle this

Reply:
already handled. forgot to remove this comment
https://github.com/gnoswap-labs/gnoswap/pull/451/commits/6de8ce8534fd7589331da2a56c25f8a61c2ef08b